tslint.json

```diff
@@ -1,42 +1,9 @@
 {
   "extends": ["tslint-config-prettier"],
-  "rulesDirectory": [
-    "node_modules/tslint-react/rules",
-    "node_modules/tslint-microsoft-contrib/",
```

Comment:
I'm an absolute novice when it comes to eslint but I can't figure out how the rules from tslint-microsoft-contrib (like react-this-binding-issue, promise-must-complete etc) get picked up in the eslint-realm. Should we keep the rules from microsoft-contrib in tslint.json in addition to our own custom rules perhaps?

Reply:
@outofambit I know you were OOO when I asked this so I'm bumping it in case you didn't see it but please don't feel stressed about it, it's not high priority

Reply:
@niik ah i think you're right in that they won't get picked up. i did some research and i think we can use https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin-tslint to move our custom tslint rules and the tslint-microsoft-contrib to be run by eslint. (they still get run by tslint at the end of the day, its just a wrapper.) this seems like a good step to keep our rules without having to run multiple linters.